Code/Framework/AzCore/AzCore/Jobs/Internal/JobManagerWorkStealing.h

```diff
@@ -21,6 +21,8 @@
 #include <AzCore/std/parallel/binary_semaphore.h>
 #include <AzCore/std/parallel/thread.h>
 
+// #define JOBMANAGER_ENABLE_STATS
+
```

Comment:
Maybe we can move this commented out #define into a CMake variable to control enabling stats or not during the project generation. Something like
`cmake -B build/windows -DO3DE_ENABLE_JOBMANAGER_STATS=TRUE` 
or something